src/domain/registration/utils.ts

```diff
@@ -200,9 +201,13 @@ export const getRegistrationWarning = (
   const attendeeCapacityUsed = isAttendeeCapacityUsed(registration);
   const waitingListCapacityUsed = isWaitingListCapacityUsed(registration);
   const freeWaitlistCapacity = getFreeWaitingListCapacity(registration);
+  const enrolmentStartTime = new Date(registration.enrolment_start_time);
 
   if (!registrationOpen) {
-    return t('signup:warnings.closed');
+    return t('signup:warnings.closed', {
+      openingDate: formatDate(enrolmentStartTime),
```

Comment:
Is there a chance that `endrolment_start_time` is not defined while `enrolment_end_time` is? I think this might then show a warning where the date and time are empty strings. For this case perhaps the older warning text without any temporal information could be shown.

Reply:
This is a problem only in theory. Only other case when registration is not open is that enrolment end date is in the past. In that case we show totally different error page. 

Anyway, I did the suggested change so this won't cause any issues in the future either.